jekyll/_cci2/rerun-failed-tests.adoc

```diff
@@ -228,6 +228,20 @@ You can also add it to your `jest.config.js` file by following these link:https:
 +
 NOTE: Ensure that you are using version 1.34.2 or later of Playwright. Earlier versions of Playwright may not output JUnit XML in a format that is compatible with this feature.
 
+[#configure-a-job-running-go-tests]
+=== Configure a job running Go tests
+
+. Modify your test command to use `circleci tests run`:
+
+```yaml
+- run:
+    command: go list ./... | circleci tests run --command "xargs gotestsum --junitfile junit.xml --format testname --" --split-by=timings --timings-type=name 
+
+- store_test_results:
+    path: junit.xml
```

Comment:
```suggestion
+
```yaml
- run:
    command: go list ./... | circleci tests run --command "xargs gotestsum --junitfile junit.xml --format testname --" --split-by=timings --timings-type=name 

- store_test_results:
    path: junit.xml
```

Reply:
Also close the ```